README.md

```diff
@@ -1,33 +1,38 @@
 # API Example
 
-Example apps using Climate's API.
+Example app using Climate's [FieldView API](https://dev.fieldview.com).
 
 ## Setup
 
 1. Install python 3.6+.
 
-    ```bash
-    # if you use Mac OS X and brew, this can be done with:
-    brew install python3
-    ```
+```bash
```

Comment:
whats the gap with using python 2.7, what happens if a user cant install version 3 due to conflicts, I ask because I've personally run into this.

Reply:
i'm not sure. thus far we haven't invested time in supporting python 2.7 but in my experience it wasn't hard to add python 3.x under the `python3` alias.